pkg/ovsdb/transact.go

```diff
@@ -45,7 +45,7 @@ const (
 )
 
 func isEqualSet(expected, actual interface{}) bool {
-	return reflect.DeepEqual(expected, actual)
+	return len(expected.(libovsdb.OvsSet).GoSet) == len(actual.(libovsdb.OvsSet).GoSet) && isIncludesSet(expected, actual)
```

Comment:
why did you repalced the golang DeepEqual implementation ?

Reply:
Because the underlying tpe of `OvsSet` is `[]interface`.
- the order is important on go slices. 
- the order is not important on `OvsSet` (because it's a set). 

Therefore, the naive `reflect.DeepEqual` will check if the slices are equal and will take into account order of the elements on the comparison.